internal/provider/waypoint/resource_waypoint_add_on_definition.go

```diff
@@ -277,19 +294,6 @@ func (r *AddOnDefinitionResource) Create(ctx context.Context, req resource.Creat
 		modelBody.ReadmeMarkdownTemplate = readmeBytes
 	}
 
-	if !plan.TerraformCloudWorkspace.IsNull() && !plan.TerraformCloudWorkspace.IsUnknown() {
```

Comment:
This was accidentally included in a merged PR so I am removing it so it's back to how it used to function.

Reply:
Are you sure you are not breaking anything by moving this change?

Reply:
Well, about a month ago this wasn't a breaking change because it was changed by a merged PR after this PR was open. I've had to rebase this PR multiple times since it's been open, which is what my comment indicates. But now, this project has done a release since this PR was open. So it's now probably a breaking change :/

I'm fine to put it back to how it was working previously if that's the intention, it's not a big deal to put it back.

Reply:
Yes please :)